prime-router/settings/organizations.yml

```diff
@@ -243,6 +255,75 @@
         port: 22
         filePath: ./upload
         credentialName: DEFAULT-SFTP
+    - name: FULL_ELR_E2E      # For the E2E UP test
+      externalName: Ignore FULL_ELR_E2E
+      organizationName: ignore
+      topic: full-elr
+      customerStatus: active
+      qualityFilter:
+        - "%messageId.exists()"
```

Comment:
I guess it just depends on what role if any you want for the quality filters in these tests. At the moment these exist simply because of the receivers I was copying from. I can modify it into something like you're suggesting and this would be an extra layer tested in these now.